definitions/json/AuthorisationCaseEvent.json

```diff
@@ -41,6 +41,18 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseEventID": "initiateCaseDraft",
```

Comment:
You can remove initiateCaseDraft we're not going to use it anymore. We're INITIATE_CASE_DRAFT going forward